scripts/bookmarklet.tmpl.js

```diff
@@ -0,0 +1,54 @@
+/* global jQuery, window, document */
+(function(window, document) {
+  var toast = document.createElement('div');
+  var b = document.getElementsByTagName('body')[0];
+  var version = null;
+
+  setStyles(toast)([
+    ['position' , 'fixed'],
+    ['top' , '20px'],
+    ['right' , '30%'],
+    ['padding' , '5px 10px'],
+    ['zIndex' , 1001],
+    ['fontSize' , '18px'],
+    ['color' , '#222'],
+    ['backgroundColor' , '#DFF2BF'],
+    ['text-align', 'center'],
+    ['border', 'solid 2px black'],
+    ['border-radius', '25px']
+  ]);
+
+  if (typeof window.R != 'undefined') {
+    return showMsg('This page already using ramda');
+  }
+  // jscs:disable
+  /* jshint ignore:start */
+  {{{ramda}}}
+  version = '{{{version}}}';
+  /* jshint ignore:end*/
+  // jscs:enable
+  showMsg('Ramda v' + version + ' loaded');
+
+  function showMsg(msg) {
+    toast.innerHTML = msg;
+    b.appendChild(toast);
+
+    window.setTimeout(function() {
+      if (typeof jQuery == 'undefined') {
+        b.removeChild(toast);
+      } else {
+        jQuery(toast).fadeOut('slow', function() {
+          jQuery(this).remove();
+        });
+      }
```

Comment:
I don't like branching based on whether jQuery is available. `b.removeChild(toast)` is fine. We needn't be fancy. :)